.drone.yml

```diff
@@ -52,8 +51,8 @@ steps:
     depends_on:
       - test
 
-  - name: test node 16
-    image: node:16
+  - name: test node 20
+    image: node:20
```

Comment:
may be use the same node version in the same pipeline

Reply:
this is expected, we want to run the tests on supported node versions (18, 20)
- the `test` step uses node 18 and runs on every push event
- the `test node 20` step uses node 20 and runs only on the tag event (before npm release)